.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,65 @@
+name: Bug Report
+description: File a bug report to help us improve
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: dropdown
+    id: software
+    attributes:
+      label: Software
+      description: What software are you referring to?
+      multiple: true
+      options:
+        - Desktop Application
```

Comment:
We should probably add the other extensions like Sublime / Raycast.

Reply:
I added the other extensions, can you double-check if I missed anything/should remove anything?

Reply:
This is not the repo where bug reports should be. That is why we have the link to our support repo in the config file.

So we don't need this bug report. We only need 1 general issue template and then we can keep the config file where people can go to report bugs or feature requests